File: feitos/2984 - Assuntos Pendentes/Main.java
import java.util.Scanner;

class Main {
    public static int assuntos(String str) {
        int cont = 0;

        for (int i = 0; i < str.length(); i++) {
            if (str.charAt(i) == '(') {
                cont++;
            } else if (str.charAt(i) == ')' && cont > 0) {
                cont--;
            }
        }

        return cont;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while(sc.hasNextLine()) {
            String str = sc.nextLine();

            int res = assuntos(str);
            if (res == 0) {
                System.out.println("Partiu RU!");
            } else {
                System.out.printf("Ainda temos %d assunto(s) pendente(s)!\n", res);
            }
        }

        sc.close();
    }
}

File: feitos/2984 - Assuntos Pendentes/main.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int assuntos(char str[]) {
    int cont = 0;

    for (int i = 0; i < strlen(str); i++) {
        if (str[i] == '(') {
            cont++;
        } else if (str[i] == ')' && cont > 0) {
            cont--;
        }
    }

    return cont;
}

int main() {
    char str[21];

    while(scanf("%s", str) != EOF) {
        int res = assuntos(str);
        if (res == 0) {
            printf("Partiu RU!\n");
        } else {
            printf("Ainda temos %d assunto(s) pendente(s)!\n", res);
        }
    }

    return 0;
}